server/views/referral-application/check-answers/show.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "../../partials/layout.njk" %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - " + step.title %}
+
+{% block content %}
+	<main class="app-container govuk-body">
```

Comment:
if we are using `{% block content %}` we shouldn't be adding a `<main>` element as `<main>` is already included in `content` and there should only be one. 
you can do `{% set mainClasses = "app-container govuk-body" %}` to ensure that the correct styles are applied